wallet/src/lib.did

```diff
@@ -164,6 +195,11 @@ service : {
   get_events: (opt record { from: opt nat32; to: opt nat32; }) -> (vec Event) query;
   get_chart: (opt record { count: opt nat32; precision: opt nat64; } ) -> (vec record { nat64; nat64; }) query;
 
+  // Children
+  list_children: () -> (vec ChildInfo) query;
```

Comment:
@adamspofford-dfinity do you think we will need to worry about paginating this list?

Reply:
@krpeacock I'm not actually sure how many canisters the average wallet manages - you would be able to answer that better than I would.